File: coders/curso_regex/conjuntos/atalhos.py
import re

texto = '1,2,3,4,5,6,a.b c?d!e[f'

print(re.findall(r'\w', texto))
print(re.findall(r'\W', texto))
print(re.findall(r'\d', texto))
print(re.findall(r'\D', texto))
print(re.findall(r'\s', texto))
print(re.findall(r'\S', texto))

File: coders/curso_regex/conjuntos/atalhos.js
const texto = '1,2,3,4,5,6,a.b c?d!e[f'

console.log(texto.match(/\d/g)) // [0-9]
console.log(texto.match(/\D/g)) // [^0-9]

console.log(texto.match(/\w/g)) // [a-zA-Z0-9_]
console.log(texto.match(/\W/g)) // [^a-zA-Z0-9_]

console.log(texto.match(/\s/g)) 
console.log(texto.match(/\S/g)) 
